docs/changelog/85326.yaml

```diff
@@ -0,0 +1,6 @@
+pr: 85326
+summary: "Log error if LDAP/AD realm has bind DN set but is missing bind password"
+area: Authentication
+type: bug
```

Comment:
I'm not sure what our convention is here: is this still a bug fix, even if it's just a log line, or should we rebrand as enhancement?